CLAUDE.MD

```
@@ -0,0 +1,163 @@
+- **Log Persistence**: `./logs` mapped to `/app/apps/backend/logs`
+
+## Development Standards
+
+### Code Style
+- **Backend**: Named exports only, `any` disallowed, `no-explicit-any: error`
+- **Frontend**: `any` allowed, `export default` for components, named exports for utilities
+- **Imports**: Builtins → externals → internals, alphabetized, blank line between groups
+- **Formatting**: Single quotes, no trailing commas, 4-space indent, 200 char width, semicolons
+
+### Naming Conventions
+| Item | Convention | Example |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)